examples/wasm-example/src/main.rs

```diff
@@ -0,0 +1,152 @@
+            }
+            Message::Line(line) => {
+                self.lines.push(line);
+            }
+            Message::Lines(mut lines) => {
+                self.lines.append(&mut lines);
+            }
+            Message::ButtonClick(button) => match button {
+                Button::SubscribeFinalized => {
+                    self.operation_title = Some("Subscribe to finalized blocks:".into());
+                    let cb: Callback<AttrValue> = ctx.link().callback(Message::Line);
+                    ctx.link()
+                        .send_future(services::subscribe_to_finalized_blocks(cb).map(|result| {
+                            let err = result.unwrap_err();
+                            Message::Error(err)
+                        }));
+                }
+                Button::FetchConstant => {
+                    self.operation_title =
+                        Some("Fetch the constant \"block_length\" of \"System\" pallet:".into());
+                    ctx.link()
+                        .send_future(services::fetch_constant_block_length().map(|result| {
+                            match result {
+                                Ok(value) => Message::Line(
+                                    format!(
+                                    "constant \"block_length\" of \"System\" pallet:\n    {value}"
+                                )
+                                    .into(),
+                                ),
+                                Err(err) => Message::Error(err),
+                            }
+                        }))
+                }
+                Button::FetchEvents => {
+                    self.operation_title = Some("Fetch events:".into());
+                    ctx.link()
+                        .send_future(services::fetch_events_dynamically().map(
+                            |result| match result {
+                                Ok(value) => {
+                                    Message::Lines(value.into_iter().map(AttrValue::from).collect())
+                                }
+                                Err(err) => Message::Error(err),
+                            },
+                        ))
+                }
+            },
+        }
+        true
+    }
+
+    fn view(&self, ctx: &Context<Self>) -> Html {
+        let reload: Callback<MouseEvent> = ctx.link().callback(|_| Message::Reload);
+
+        let subscribe_finalized = ctx
+            .link()
+            .callback(|_| Message::ButtonClick(Button::SubscribeFinalized));
+
+        let fetch_constant = ctx
+            .link()
+            .callback(|_| Message::ButtonClick(Button::FetchConstant));
+
+        let fetch_events = ctx
+            .link()
+            .callback(|_| Message::ButtonClick(Button::FetchEvents));
+
+        html! {
+            <div>
+                if let Some(operation_title) = &self.operation_title{
+                    <button onclick={reload}>{"🡄 Back"}</button>
```

Comment:
nit: the prefixed characters do not render properly on my machine, could we change it to something else?
Maybe plain `Back` or with some other chars?